lang/python/setup_pip.py

```diff
@@ -231,7 +231,8 @@ def get_library_dirs():
 inc_paths = [ os.path.join(build_dir, 'include'), os.path.join(build_dir, 'config'), build_dir, '.' ]
 lib_paths = [ '.' ]
 
-check_needed_dependencies(builtins, inc_paths, lib_paths)
+# Find and locate dependencies to be linked - optional
+# check_needed_dependencies(builtins, inc_paths, lib_paths)
```

Comment:
I think in this file we should remove `builtins`, `builtin_names`, `builtin_libraries` (and code that uses them, like lines 226-227 and line 221.  Otherwise there's some indication that we support these builtin compressors with the public python installation, and we really don't anymore.

Also, it won't work (I think) if we leave the code as is.  Part of what `check_needed_dependencies` did was find the installation for the library, and add the right information to inc_paths and lib_paths, which the compiler will use to compile and link.  If we don't do that, I think we are still telling cmake to build with the 3 builtins, but we aren't telling it everything it needs to compile.  I may be wrong, you can try it out.  Which leads to testing....

Before committing changes to this script, we should make sure it works so we aren't surprised at release time.  The recipe for testing has been here:  https://github.com/wiredtiger/wiredtiger/wiki/HOWTO%3AUploadAReleaseForPythonInstalls

See sections "Preparing a release for upload" and "Testing the Release".   @lukech, can you confirm that this is still the procedure in use?  The procedure basically packages the release in the form it _would_ be uploaded to pypi, and then (without uploading/downloading), uses that package to install in a new "virtual environment".

Reply:
@ddanderson 

> can you confirm that this is still the procedure in use? 

I can confirm we still use that procedure. 

> I'd rather have them try out WT without complications even if it means they can't have access to compression (which is a somewhat advanced feature anyway).

Sounds reasonable to me.

Reply:
Thanks Don! I have ran through the testing process and the tests currently runs successfully without any issues after my changes, so it's safe to remove the the builtins. In terms of the zlib guidance, there are currently some messages guiding the users, should I add some more instructions or is the current ones sufficient?